clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -1252,9 +1252,13 @@ private String getUpstreamKafkaUrlFromKafkaValue(KafkaMessageEnvelope kafkaValue
       throw new VeniceException(
           String.format(
               "No Kafka cluster ID found in the cluster ID to Kafka URL map. "
-                  + "Got cluster ID %d and ID to cluster URL map %s",
+                  + "Got cluster ID %d and ID to cluster URL map %s. "
+                  + "Message type: %d; ProducerMetadata: %s; LeaderMetadataFooter: %s",
```

Comment:
I think we need a few more info:
1. which topic is this message from
2. the offset of this message
3. which Kafka cluster is it from (this one might be difficult; if so, it's good to have the above two info first)

Reply:
Sure, I'll change the function signature so I can add that.

Reply:
Done in the latest commit.

Reply:
I was missing the third one, but actually, it is easy to add as well, so I did in the latest commit. All 3 infos you asked for are included now.